test/freenet/client/filter/ContentFilterTest.java

```diff
@@ -258,6 +266,12 @@ private String headFilter(String data) throws Exception {
 		return s;
 	}
 
+	@Test
+	public void testMetaCharset() throws IOException {
+		assertEquals(META_CHARSET, META_CHARSET);
```

Comment:
I’m gonna need an explanation for these two lines, please… 🙂

Reply:
Blindness! Utter Blindness! ← sadly that … thank you for your review! I’ll rework this.

Reply:
It wasn’t WIP until someone detected the utter blindness that hat stricken me :)